.travis.yml

```diff
@@ -59,7 +59,7 @@ script:
     rustup component add clippy;
     cargo clippy --release;
     fi
-  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.14" ] && ! [ "$ARM" == "1" ]; then
+  - if ! [ "$TRAVIS_RUST_VERSION" == "nightly" ] && ! [ "$TRAVIS_RUST_VERSION" == "beta" ] && [ "$GTK" == "3.14" ] && ! [ "$ARM" == "1" ]; then
```

Comment:
Hmm why not `== stable`? Hrm, and actually add a stable job to travis in addition to the lower-bound 1.36 job :)

Reply:
Sounded simpler that way. I don't want yet another special case.

Reply:
We should have a stable job too though, that is always running with latest stable instead of fixed 1.36

Reply:
This is a good argument. Should I run it on both OSX and linux or just linux is fine?

Reply:
all